funtofem/model/_base.py

```diff
@@ -199,9 +199,10 @@ def verify_vartype(self, vartype):
             "rigid_motion",
             "shape",
             "controls",
+            "thermal",
         ]:
             print(
-                "Warning: vartype specified is not a recognized variable type",
+                f'Warning: vartype "{vartype}" is not a recognized variable type',
```

Comment:
Can you make an attribute VAR_TYPES = ["structural", "aerodynamic", "rigid_motion", "shape", "controls", "thermal"] above the __init__ constructor. This way people can easily see the available types at the top of the file. Then you can do:

```
if not vartype in self.VAR_TYPES:
    print(f"Warning ... etc")
```

Reply:
Good idea, I implemented it in the latest commit